### ISSUE TITLE
`x.py setup` should set `changelog-seen`

### ISSUE THREAD

**Issue by jyn514** [rust-lang/rust#77572]
New contributors haven't use `x.py` before, so there's no reason for them to read the changelog: nothing has changed.

```
$ x.py setup
Updating only changed submodules
Submodules updated in 0.02 seconds
    Finished dev [unoptimized + debuginfo] target(s) in 0.09s
Welcome to the Rust project! What do you want to do with x.py?
a) Contribute to the standard library
b) Contribute to the compiler
c) Contribute to the compiler, and also modify LLVM or codegen
d) Install Rust from source
Please choose one (a/b/c/d): a
warning: x.py has made several changes recently you may want to look at
help: consider looking at the changes in `src/bootstrap/CHANGELOG.md`
note: to silence this warning, add `changelog-seen = 2` at the top of `config.toml`
`x.py` will now use the configuration at /home/joshua/rustc2/src/bootstrap/defaults/config.toml.library
To get started, try one of the following commands:
- `x.py check`
- `x.py build`
- `x.py test library/std`
- `x.py doc`
For more suggestions, see https://rustc-dev-guide.rust-lang.org/building/suggested.html
warning: x.py has made several changes recently you may want to look at
help: consider looking at the changes in `src/bootstrap/CHANGELOG.md`
note: to silence this warning, add `changelog-seen = 2` at the top of `config.toml`
note: this message was printed twice to make it more likely to be seen
$ x.py check
Updating only changed submodules
Submodules updated in 0.01 seconds
    Finished dev [unoptimized + debuginfo] target(s) in 0.09s
warning: x.py has made several changes recently you may want to look at
help: consider looking at the changes in `src/bootstrap/CHANGELOG.md`
note: to silence this warning, add `changelog-seen = 2` at the top of `config.toml`
Checking std artifacts (x86_64-unknown-linux-gnu -> x86_64-unknown-linux-gnu)
```

**Comment by winnayx** [comment]
I can tackle this.

**Comment by jyn514** [comment]
@winnayx that would be great! To get started, take a look at `src/bootstrap/setup.rs`. In addition to adding `profile = "..."`, also add `changelog-seen = VERSION`. You'll have to move `VERSION` from `bootstrap/bin/main.rs` to `bootstrap/lib.rs` so you can access it in `setup`.

You should also double check that the changelog warning isn't printed during the `setup` command itself. I'm not sure exactly how to do that without reading the code, but I expect it to be a simple `if cmd != "setup"` or something like that.
